### PR TITLE
ACMS-1483: Headless drush command test.

### PR DESCRIPTION
**Pull request by apathak18**
**Motivation**

Fixes ACMS-1483

**Proposed changes**
New test case for headless drush commands.

### CODE REVIEW

**Review by rajeshreeputra — APPROVED** [review]
LGTM

File: modules/acquia_cms_headless/tests/src/Functional/HeadlessDrushCommandsTest.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    $this->drush('acms:headless:new-nextjs', [], $options);
+    $newNextJsData = $this->getErrorOutputAsList();
+    $this->assertEquals("Use these environment variables for your Next.js application. Place them in your .env file:", str_replace("[notice] ", "", $newNextJsData[0]));
+    $this->assertEquals("NEXT_PUBLIC_DRUPAL_BASE_URL=http://default", $newNextJsData[1]);
+    $this->assertEquals("NEXT_IMAGE_DOMAIN=default", $newNextJsData[2]);
+    $this->assertEquals("DRUPAL_SITE_ID=headless_site", $newNextJsData[3]);
+    $this->assertEquals("DRUPAL_FRONT_PAGE=/user/login", $newNextJsData[4]);
+    $this->assertStringContainsString("DRUPAL_PREVIEW_SECRET", $newNextJsData[5]);
+    $this->assertStringContainsString("DRUPAL_CLIENT_ID", $newNextJsData[6]);
+    $this->assertStringContainsString("DRUPAL_CLIENT_SECRET", $newNextJsData[7]);
```
Comment (chandan-singh7929) [inline modules/acquia_cms_headless/tests/src/Functional/HeadlessDrushCommandsTest.php:50]: to be precise we should use:
```
$this->assertStringStartsWith("DRUPAL_PREVIEW_SECRET", $newNextJsData[5]);
$this->assertStringStartsWith("DRUPAL_CLIENT_ID", $newNextJsData[6]);
$this->assertStringStartsWith("DRUPAL_CLIENT_SECRET", $newNextJsData[7]);
$this->assertStringStartsWith("DRUPAL_CLIENT_SECRET", $newNextJsData[7]);
```

**Review by chandan-singh7929 — APPROVED** [review]
Overall it looks good to me.

File: modules/acquia_cms_headless/tests/src/Functional/HeadlessDrushCommandsTest.php (hunk 1 above)
Comment (apathak18) [reply modules/acquia_cms_headless/tests/src/Functional/HeadlessDrushCommandsTest.php]: Done

**Review by vishalkhode1 — APPROVED** [review]

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**